prow/apis/prowjobs/v1/types.go

```diff
@@ -171,6 +171,10 @@ type GitHubTeamSlug struct {
 	Org  string `json:"org"`
 }
 
+// RerunAuthConfigs represents the configs for rerun authorization in Deck.
+// Use `org/repo`, `org` or `*` as key and a `RerunAuthConfig` struct as value.
+type RerunAuthConfigs map[string]RerunAuthConfig
```

Comment:
Why is this part of the CRD definition rather than the config in prow/config? Its only used in the prow/config package or did I miss something?

Reply:
No particular reason, other that oversight on directory structure. I will move this type and its methods into `prow/config`.

Reply:
Updated